lua/fishing_mod/cl_networking.lua

```diff
@@ -152,12 +152,12 @@ net.Receive("Fishingmod:Bait", function()
 end)
 
 hook.Add("Tick", "Fishingmod.CleanInfo:Tick", function()
-	for key, value in pairs(fishingmod.InfoTable.Catch) do
+	for key, value in ipairs(fishingmod.InfoTable.Catch) do
 		if not IsValid(Entity(key)) then
 			fishingmod.InfoTable.Catch[key] = nil
 		end
 	end
-	for key, value in pairs(fishingmod.InfoTable.Bait) do
+	for key, value in ipairs(fishingmod.InfoTable.Bait) do
```

Comment:
fishingmod.InfoTable.Bait is not sequential

As ipairs states on the wiki
`This will only iterate though numerical keys, and these must also be sequential; starting at 1 with no gaps.`

You can see the way entries are added [here](https://github.com/Metastruct/fishingmod/blob/master/lua/fishing_mod/cl_networking.lua#L129-L138)